src/OpenTelemetry/Resources/Resource.cs

```diff
@@ -57,11 +67,31 @@ public Resource(IEnumerable<KeyValuePair<string, object>> attributes)
         /// </summary>
         public static Resource Empty { get; } = new Resource(Enumerable.Empty<KeyValuePair<string, object>>());
 
+        /// <summary>
+        /// Gets the dafault Resource with telemetry sdk attributes set.
+        /// </summary>
+        public static Resource Default { get; } = new Resource(new List<KeyValuePair<string, object>>
```

Comment:
Why is Default only having `Telemetry SDK` part?

Reply:
I think the default should be "TelemetrySDK, plus anything coming from the ENV variables."

Reply:
Yup agreed. Made that change.